README.md

```
@@ -74,16 +101,28 @@ GOOGLE_CLIENT_SECRET=REPLACE_WITH_YOUR_GOOGLE_CLIENT_SECRET
+# Run migrations with uv run
+uv run alembic upgrade head
+
+# Navigate back and start server
+cd ..
+uv run uvicorn main:app --reload --host 0.0.0.0 --port 8000
+```
+
 
-# Start the backend server
-uvicorn main:app --reload --host 0.0.0.0 --port 8000
 ```
```

markdownlint-cli2 (0.18.1)

Fenced code blocks should have a language specified

(MD040, fenced-code-language)